packages/scripts/scripts/test-e2e.js

```diff
@@ -12,8 +12,7 @@ process.on( 'unhandledRejection', ( err ) => {
 /**
  * External dependencies
  */
-/* eslint-disable jest/no-jest-import */
-const jest = require( 'jest' );
+const jest = require( 'jest' ); // eslint-disable-line jest/no-jest-import
```

Comment:
The coding standards mention that comments should preferably not be inlined and I think there shouldn't be an exception for eslint as `eslint-disable-next-line` can be used here.

Reply:
For my personal curiousity, can you point me to the relevant section in the coding standard that mentions that?

Reply:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#comments

Reply:
Thanks! Sounds like we could explore enabling the `no-inline-comments` rule. Will try to look into it.